plugins/processors/starlark/README.md

```diff
@@ -182,6 +189,29 @@ def apply(metric):
 def failing(metric):
     json.decode("non-json-content")
 ```
+**How to reuse the same script but with different parameters?**
+
+In case you have a generic script that you would like to reuse for different instances of the plugin, you can use constants as input parameters of your script.
+
+So for example, assuming that you have the next configuration:
+
+```toml
+[[processors.starlark]]
+  script = "/usr/local/bin/myscript.star"
+
+  [processors.starlark.constants]
+    somecustomnum = 10
+    somecustomstr = "mycustomfield"
+```
+
+Your script could then use the constants defined in the configuration as next:
```

Comment:
grammar nit-pick: replace "the configuration as next" with "the following configuration"